example/src/main/scala/org/apache/pekko/persistence/cassandra/example/EventProcessorStream.scala

```diff
@@ -85,7 +85,7 @@ class EventProcessorStream[Event: ClassTag](
   private def readOffset(): Future[Offset] = {
     session
       .selectOne(
-        "SELECT timeUuidOffset FROM akka.offsetStore WHERE eventProcessorId = ? AND tag = ?",
+        "SELECT timeUuidOffset FROM pekko.offsetStore WHERE eventProcessorId = ? AND tag = ?",
```

Comment:
Not strictly related to review, but I think we probably need to provide an SQL migration script. Should be discussed on mailing list @pjfanning

Reply:
I think we can all agree that replacing `akka` in the schema name makes sense. We should at least log an issue to track what to do about notifying users about the migration SQL they need to run.

Reply:
Oh definitely we should replace `akka`, but this is as you said as a bare minimum we should at least notify users because it is a breaking change.

Reply:
Agreed, we will need a good migration guide and tools if possible

Reply:
I've created an issue to track this: https://github.com/apache/incubator-pekko-persistence-cassandra/issues/32